numpy/ma/core.py

```diff
@@ -1123,7 +1129,7 @@ def __call__(self, a, b, *args, **kwargs):
         # Apply the domain
         domain = ufunc_domain.get(self.f, None)
         if domain is not None:
-            m |= filled(domain(da, db), True)
+            m |= domain(da, db)
```

Comment:
The call to `filled` has no effect since da and db have no masked values (they are just the data arrays of a and b).